src/FE/src/app/pages/dashboard-page/dashboard-page.component.html

```diff
@@ -120,19 +121,23 @@ <h2>{{ selectedCountry?.name }}</h2>
                 </div>
               </mat-list-item>
             }
-            <mat-divider></mat-divider>
-            <div mat-subheader>Country Total</div>
-            <mat-list-item>
-              <div matListItemTitle>Total cost</div>
-              <div matListItemLine>{{ this.currencyService.getSymbol(localStorage.getItem("selectedCurrency")!) }} {{ selectedCountry?.totalCost | number: '1.0-2' }}</div>
-            </mat-list-item>
-            <mat-list-item>
-              <div matListItemTitle>Total price</div>
-              <div matListItemLine>{{ this.currencyService.getSymbol(localStorage.getItem("selectedCurrency")!) }} {{ selectedCountry?.totalPrice | number: '1.0-2' }}</div>
-            </mat-list-item>
+            <mat-divider/>
+            <div mat-subheader class="selectedCountryUndertitle">{{ "PROJECT_DAY_RATE" | translate }}</div>
+            @for (countryData of selectedCountry!.projects; track countryData) {
+              <mat-list-item>
+                <div matListItemTitle>{{ countryData.name }}</div>
+                <div
+                  matListItemLine>{{ this.currencyService.getSymbol(localStorage.getItem("selectedCurrency")!) }} {{ countryData.dayRate | number: '1.0-2' }}
```

Comment:
er det muligt, at dele denne i to linjer? uden det ødelægger viewet. men bare så vi ikke har kilometer lange linjer